### PR TITLE
enh(release): Add Centreon Web 21.10.6 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [x] 21.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 1]
```diff
@@ -17,6 +17,36 @@ If you have feature requests or want to report a bug, please go to our
...
+
+- [API] Fixed an issue in the `icons` API endpoint that always returned 0 for total number of results
+- [Banner] Fixed display of empty skeleton
+- [Charts] Fixed slowdown in graphics display
+- [Configuration] Fixed an issue that caused the export of the poller configuration files to fail when a disabled host template was used
+- [Configuration] Fixed checkbox selection after enabling/disabling a contact via icons
+- [Core] Proxy settings were saved with empty parameters
+- [Install] Fixed database user creation with remote DBMS
+- [Monitoring] Fixed display of acknowledgement information in legacy Resources Status pages
+- [Monitoring] fixed relation issue for recurrent downtimes
```
Comment (cg-tw) [inline versioned_docs/version-21.10/releases/centreon-core.md:32]:
```suggestion
- [Monitoring] Fixed relation issue for recurrent downtimes
```

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 2]
```diff
@@ -17,6 +17,36 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Resources Status] Fixed Hard/Soft translation
+- [Resources Status] Fixed monitoring command not displayed in Resources Status Details panel
+- [UX] Fixed display of date with UTC timezone in datepickers
+- [UX] Improved interface response time if CEIP is enabled but the browser does not have internet access
+
+#### Security Fixes
+
+- [Apache] Fixed cookies with missing or contradictory properties
+- [Apache] Improve Apache configuration to enable HSTS
+- [Configuration] Fixed a SQL injection issue in Configuration > Poller > Resources
```
Comment (cg-tw) [inline versioned_docs/version-21.10/releases/centreon-core.md:45]:
```suggestion
- [Configuration] Fixed an SQL injection issue in Configuration > Poller > Resources
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md [hunk 3]
```diff
@@ -16,6 +16,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [UX] Improved interface response time if CEIP is enabled but the browser does not have internet access
+
+#### Security Fixes
+
+- [Apache] Fixed cookies with missing or contradictory properties
+- [Apache] Improve Apache configuration to enable HSTS
+- [Configuration] Fixed a SQL injection issue in Configuration > Poller > Resources
+- [Core] Passwords are now obfuscated in the page's HTML source
+- [Core] Replace Math.random by Crypto JS API
+- [PHP] Disabling allow_url_fopen in PHP
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md:47]:
```suggestion
- [PHP] Fixing an issue by disabling allow_url_fopen in PHP
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 4]
```diff
@@ -17,6 +17,36 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Resources Status] Fixed display of acknowledgements comments
+- [Resources Status] Fixed Hard/Soft translation
+- [Resources Status] Fixed monitoring command not displayed in Resources Status Details panel
+- [UX] Fixed display of date with UTC timezone in datepickers
+- [UX] Improved interface response time if CEIP is enabled but the browser does not have internet access
+
+#### Security Fixes
+
+- [Apache] Fixed cookies with missing or contradictory properties
+- [Apache] Improve Apache configuration to enable HSTS
```
Comment (smau-centreon) [inline versioned_docs/version-21.10/releases/centreon-core.md:44]:
```suggestion
- [Apache] HTTPS Apache configuration now includes HSTS
```

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 5]
```diff
@@ -17,6 +17,36 @@ If you have feature requests or want to report a bug, please go to our
...
+### 21.10.6
+
+#### Bug Fixes
+
+- [API] Fixed an issue in the `icons` API endpoint that always returned 0 for total number of results
+- [Banner] Fixed display of empty skeleton
+- [Charts] Fixed slowdown in graphics display
+- [Configuration] Fixed an issue that caused the export of the poller configuration files to fail when a disabled host template was used
+- [Configuration] Fixed checkbox selection after enabling/disabling a contact via icons
+- [Core] Proxy settings were saved with empty parameters
```
Comment (smau-centreon) [inline versioned_docs/version-21.10/releases/centreon-core.md:29]:
```suggestion
- [Core] Fixed an issue where proxy settings were saved with empty parameters
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by mdebauchez — CHANGES_REQUESTED** [review]
Shouldn't we add the following tickets:
- MON-12744 : Date/Time on the top-right doesn't respect user configuration
- MON-12494 : CEIP - Add IT-100 license type
- MON-12337 : [CEIP] Add a unique identifier for css objects in Resources Status listing
- MON-12448 : Synchronise update scripts between 21.10.x and develop
- MON-12129 : [CEIP] Identify Centreon Cloud platform

**Review by mdebauchez — DISMISSED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md [hunk 6]
```diff
@@ -16,6 +16,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 21.10.6
+
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md:20]:
```suggestion

Release date: `May 2, 2022`

```

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 7]
```diff
@@ -17,6 +17,36 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.10.6
+
```
Comment (cg-tw) [inline versioned_docs/version-21.10/releases/centreon-core.md:21]:
```suggestion


Release date: `May 2, 2022`

```

**Review by cg-tw — APPROVED** [review]

**Review by mdebauchez — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**